src/components/App.tsx

```diff
@@ -44,6 +45,10 @@ const App: React.FC<AppProps> = (props) => {
     return <StreamContainer />
   }
 
+  if (small || mobile) {
+    return <MobileContainer />
+  }
```

Comment:
Now that we manage the mobile case from here, I think you should also remove the throw in unity-renderer to prevent unexpected behaviors from kernel and renderer

Reply:
I'm counting on that throw to stop unity. How do you think that could affect?

Reply:
@menduz ?

Reply:
We are thinking on removing that throw at some point. Shouldn't be a blocker for this PR